docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc

```diff
@@ -92,6 +91,63 @@ WARNING: The reset will affect all widgets and the general page settings of the
 To reset all pages and remove all user created pages clear the browser's local storage for the monitoring console page.
 
 
+[[monitoring-console-page-sync]]
+==== Page Synchronisation
+Monitoring Console uses the browser's local storage to store the page configuration (which includes the widgets placed on a page).
+This means by default changes to page configurations do not affect other users.
+To make pages available to others and to start from a common base configuration pages are explicitly synchronised with a persistent server page configuration.
+
+The chosen role controls the behaviour and available options for the page synchronisation.
+It is the _Administrator_'s role that is meant to manage the persistent server page configuration.
+The _Administrator_ pushes local changes to the server to update the base configuration.
+_User_s and _Guest_s pull the server configuration to update their local configuration with the base.
+The control remains with each user.
+
+The _Page Synchronisation_ dialogue shown when entering the console UI allows the user to pick the pages that should be updated (pulled).
+
+image::monitoring-console/mc_dia_pages.png[Page Synchronisation]
+
+Based on the role the pages to synchronise are pre-selected.
+
+* _Guest_: by default updates all pages available remotely unless they are already up to date
+* _Administrator_ and _User_: by default updates all pages available remotely unless a page has been locally changed
+
+The newest version is marked by a green background for easier orientation.
+
+This dialogue can also be opened from the _Global_ settings by clicking the _Update Local Pages_ button shown _Page Sync_ property.
+
+image::monitoring-console/mc_setting_global_rolesync.png[Global Page Synchronisation Settings]
+
+_Administrator_s also find a button to _Update Remote Pages_ with any changes applied to local changes.
```

Comment:
Again, italics are going weird here. Might just be GitHub preview.

Reply:
```suggestion
__Administrator__s also find a button to _Update Remote Pages_ with any changes applied to local changes.
```